monitor/linuxmonitor/processor.go

```diff
@@ -128,7 +129,8 @@ func (s *LinuxProcessor) Destroy(eventInfo *rpcmonitor.EventInfo) error {
 	if err != nil {
 		return err
 	}
-
+	strtokens := strings.Split(contextID, "/")
+	contextID = strtokens[len(strtokens)-1]
```

Comment:
panics is contextID has no /

Reply:
+1. Need the safe routine as we do this in so many places.